cmd/kube-scheduler/app/options/deprecated.go

```diff
@@ -68,6 +67,8 @@ func (o *DeprecatedOptions) AddFlags(fs *pflag.FlagSet, cfg *componentconfig.Kub
 	fs.MarkDeprecated("hard-pod-affinity-symmetric-weight", "This option was moved to the policy configuration file")
 	fs.StringVar(&cfg.FailureDomains, "failure-domains", cfg.FailureDomains, "Indicate the \"all topologies\" set for an empty topologyKey when it's used for PreferredDuringScheduling pod anti-affinity.")
 	fs.MarkDeprecated("failure-domains", "Doesn't have any effect. Will be removed in future version.")
+	fs.Int32Var(&cfg.PercentageOfNodesToScore, "percentage-of-nodes-to-score", cfg.PercentageOfNodesToScore,
+		"the scheduler stops finding feasible nodes once it finds this percentage of all nodes feasible. If your cluster size is less several hundred nodes, you should probably not set this flag; it won't make much difference anyway.")
```

Comment:
This probably sounds vague, probably `if your cluster size is less than 100 nodes` or something more concrete would help but I know that this is first time we are doing.

Reply:
The point of this description is to convey the message that users shouldn't worry about this flag if their cluster size is not in the order of thousand nodes. I intentionally didn't want to say 100.